micro-airtable-api.test.js

```diff
@@ -0,0 +1,77 @@
+const http = require('http');
```

Comment:
Mind renaming this file back to `index.test.js`? It's a standard Jest practice to name the test files the same as the files they're testing. Since we're testing `index.js`, it should be named `index.test.js` too.

Reply:
Oh, one more thing: I noticed a few formatting oddities. Mind running the file through prettier? We'll need to update the `package.json` scripts to include this file.